nni/tools/jupyter_extension/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import proxy
+
+load_jupyter_server_extension = proxy.setup
+_load_jupyter_server_extension = proxy.setup
```

Comment:
why `load_jupyter_server_extension` and `_load_jupyter_server_extension`, and where we use these?

Reply:
These are meta-info of jupyter extension, for different jupyter versions.

Reply:
got it.